clang/lib/Sema/SemaType.cpp

```diff
@@ -1575,7 +1589,7 @@ static QualType ConvertDeclSpecToType(TypeProcessingState &state) {
   case DeclSpec::TST_decimal64:    // _Decimal64
   case DeclSpec::TST_decimal128:   // _Decimal128
     S.Diag(DS.getTypeSpecTypeLoc(), diag::err_decimal_unsupported);
```

Comment:
This diag can be removed, right?

Reply:
Yes, but not yet. I have other changes that I haven't posted for review yet that will address this.

Reply:
Are you going to add the changes to this patch or create another one?

Reply:
That will happen in a different PR as it is tied to plumbing for new command line options to enable/disable DFP language support and to select a DFP run-time library.